Dockerfile

```diff
@@ -23,7 +23,7 @@ EXPOSE 5050
 ENV GIT_HASH $GIT_HASH
 ENV HELM_DEBUG 1
 # WARNING: If you are changing any versions here, update it in the reference.conf
-ENV TERRA_APP_SETUP_VERSION 0.0.2
+ENV TERRA_APP_SETUP_VERSION 0.0.3
```

Comment:
Bumped the setup chart to add Azure support. See: https://github.com/DataBiosphere/terra-app/pull/21

The change should be backwards compatible with GKE, will verify tests.